lib/src/screens/initial_setup/widgets/initial_setup_welcome.dart

```diff
@@ -99,75 +97,81 @@ SizedBox initialSetupWelcome(BuildContext context) {
 /// This function constructs a `Row` widget designed to display a single piece
 /// of information in a profile UI. It is primarily used for laying out text-based
 /// information such as names, titles, or other key details in the profile section.
+///
+/// // comment out the following function as it is not used in the current version
```

Comment:
How does this affect the doc string? It may look somewhat odd in the generated docs?